pyproject.toml

```diff
@@ -1,3 +1,53 @@
+[project]
+name = "ankihub_addon"
+description = "AnkiHub add-on for Anki"
+readme = "README.md"
+requires-python = ">=3.9"
+version = "2025.07.04"
+dependencies = []
+
+[tool.setuptools]
+package-dir = { "ankihub_addon" = "ankihub" }
+
+[dependency-groups]
+production = [
+  "aqt[qt6]==24.11",
+]
+tests = [
+  "aqt[qt5]==24.11",
```

Comment:
out of curiosity, why do we test on a different Qt version than we use in prod?

Reply:
qt6 fails with "Fatal Python error: Segmentation fault" on CI for some reason.